### ISSUE TITLE
Push.create raises exception in all cases when permission denied

### ISSUE THREAD

**Issue by joallard** [Nickersoft/push.js#95]
The promise already says whether it's accepted or rejected, I'd expect an invocation that doesn't necessarily raise an exception when permission is denied.

**Comment by realfresh** [comment]
Agreed, this just raised an exception in a production application for a browser it didn't support casuing it to hang
